http.ts

```diff
@@ -23,7 +23,7 @@ Serializer.add(
 			? {
 					...message,
 					header: { ...message.header, link: [message.body.cursor, 'rel="next"'] },
-					body: JSON.stringify((({ cursor, ...data }) => data)(message.body)),
+					body: JSON.stringify([...message.body]),
```

Comment:
How does the cursor get removed?

Reply:
By deconstructing the array and rebuilding it, arrays have no cursor property.
![image](https://github.com/utily/cloudly-http/assets/79835961/71a59226-d3ac-4fba-8839-6bf33f14726f)